NAM/activations.cpp

```diff
@@ -6,6 +6,8 @@ activations::ActivationHardTanh _HARD_TANH = activations::ActivationHardTanh();
 activations::ActivationReLU _RELU = activations::ActivationReLU();
 activations::ActivationSigmoid _SIGMOID = activations::ActivationSigmoid();
 
+bool activations::Activation::using_fast_tanh = false;
```

Comment:
I guess this is a separate but non-conflicting way to do this, then? FWIW, I think I'm happy to have it be global like this 👍🏻

Reply:
> I guess this is a separate but non-conflicting way to do this, then?

Because the LSTM code calls the activation functions directly (since it is operating on individual weights), it needed a flag to key off of.